common/src/main/java/org/apache/drill/common/expression/SchemaPath.java

```diff
@@ -54,15 +50,16 @@ public static SchemaPath getSimplePath(String name) {
   }
 
   public static SchemaPath getCompoundPath(String... strings) {
```

Comment:
The change I made removes the side effect. Do you mean I should add a description of the method's purpose ?